README.md

```diff
@@ -1,29 +1,55 @@
-# Morello-API
-The interface for executing binaries on morello board and self that returns output which can be processed by the client.
+# morello-api
 
+## Description
 
-## Local
-Update your local.json to contain a variable for your QEMU morello-purecap system **morello.address**. File mmight not be created just yet, so you would need to create one in `./config/` folder and add a new properrty, please look at the `default.json` on how other variables are configured. If you would like to connect using, then you would need to run quemu with `--run/ssh-forwarding-port=2222` switch so it woould look something like -> `./cheribuild.py run-morello-purecap --run/ephemeral --run/ssh-forwarding-port=2222 -d` and then connect `ssh -p 2222 localhost`. For uploading files `scp -v -P 2222 <FILE> localhost:/tmp`.
+A `Node.js` API for executing binaries on Morello and `aarch64` architecture using a QEMU instance of the [Morello SDK](https://github.com/CTSRD-CHERI/cheribuild). The API copies binaries to QEMU, executes them and returns the output.
+
+## Configuration
+
+Use a `.env` at root of the repository to set values for the environment variables defined in [custom-environment-variables.json](./config/custom-environment-variables.json).
+
+| variable         | required |   default   | description                                                                          |
+| :--------------- | :------: | :---------: | :----------------------------------------------------------------------------------- |
+| PORT             |    N     |   `3000`    | The port for the API to listen on                                                    |
+| LOG_LEVEL        |    N     |   `debug`   | Logging level. Valid values are [`trace`, `debug`, `info`, `warn`, `error`, `fatal`] |
+| MORELLO_HOST     |    N     | `127.0.0.1` | Morello SDK QEMU host name                                                           |
```

Comment:
Not necessarily QEMU hostname.  Just "Morello hostname" is fine here

Reply:
done and removed references to QEMU in couple of other places